cluster-sync/sync.sh

```diff
@@ -190,6 +209,7 @@ if [[ ! -z "$UPGRADE_FROM" ]]; then
     fi
     echo "Currently at version: $VERSION"
     wait_cdi_pods_updated
+    setup_for_upgrade_testing
```

Comment:
is this supposed to be run in a loop?

Reply:
nope - we want to run it once to setup an environment with DVs from one of the versions, and if we do multiple upgrades before running any tests then not set it up again

Reply:
so the function is run inside a loop (lines 176-213) but because of the condition inside it only run on the first time

so maybe it would be better to call it before the loop? or after the loop?

Reply:
I wanted to install DVs in the first version we upgrade from, which is the oldest, so after the first loop run.
before: no CDI version installed
after: fairly recent CDI version

Reply:
in that case, it is good, 
maybe a simple comment for anyone wanting to update/remove it in the future